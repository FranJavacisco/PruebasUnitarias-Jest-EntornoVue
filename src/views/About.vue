<template>
    <div>
        <h1>About Page</h1>
        <p>Esta es la p√°gina About.</p>
    </div>
</template>
<script>
export default {
    name: 'About'
}
</script>