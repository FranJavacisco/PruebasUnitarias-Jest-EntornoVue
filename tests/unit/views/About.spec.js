import { shallowMount } from '@vue/test-utils';
import About from '@/views/About.vue';

describe('About.vue', () => {
    it('matches the snapshot', () => {
        const wrapper = shallowMount(About);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
