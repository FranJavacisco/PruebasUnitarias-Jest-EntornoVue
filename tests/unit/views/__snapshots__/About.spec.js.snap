// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue matches the snapshot 1`] = `
<div>
  <h1>About Page</h1>
  <p>Esta es la p√°gina About.</p>
</div>
`;
